README.md

```diff
@@ -7,3 +7,5 @@ Linux Build Status: [![Build Status](https://travis-ci.org/csm10495/cNVMe.svg?br
 
 Windows Build Status: [![Build status](https://ci.appveyor.com/api/projects/status/7fovagftoeoahgup/branch/master?svg=true)](https://ci.appveyor.com/project/csm10495/cnvme/branch/master)
```

Comment:
Can't currently port appveyor/travis ci jobs to the Intel repo. ... I have an internal Jira for that (OTCIS-13133)... Also see #3.